app/facility/[id]/applications/jobPost/[jobPostId]/page.tsx

```diff
@@ -0,0 +1,69 @@
+import { shiftProvider } from "@/app/providers/shiftProvider";
+import ShiftsManager from "../../../job-post/components/shiftsManager";
+
+interface ShiftStaffProfile {
+  firstname: string;
+  lastname: string;
+}
+
+interface FetchedShift {
+  id: string;
+  jobPostId: string;
+  staffProfileId: string;
+  start: Date;
+  end: Date;
+  status: "Scheduled" | "InProgress" | "OnBreak" | "Completed" | "Confirmed";
+  clockInTime?: Date;
+  clockOutTime?: Date;
+  createdAt: Date;
+  updatedAt: Date;
+  staffProfile: ShiftStaffProfile;
+}
+
+export interface Shift {
+  id: string;
+  jobPostId: string;
+  staffProfileId: string;
+  start: Date;
+  end: Date;
+  status: "Scheduled" | "InProgress" | "OnBreak" | "Completed" | "Confirmed";
```

Comment:
Isn't it easier to just import the Status enum from prisma/client and use that as the type here, so we're consistent between frontend and backend?